http-api/src/main/java/io/avaje/http/api/PathVariable.java

```diff
@@ -0,0 +1,21 @@
+package io.avaje.http.api;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/** Marks a method parameter to be a path variable. */
```

Comment:
Javadoc needs to include some commentary on when this is needed etx.

Reply:
Hmm, not convinced this should exist at all. There is some design decision made here that isn't clear - no test uses PathVariable etc.